fleet-examples/gitrepos/metallb-suse-edge-gitrepo.yaml

```diff
@@ -1,7 +1,7 @@
 kind: GitRepo
 apiVersion: fleet.cattle.io/v1alpha1
 metadata:
-  name: metallb
+  name: metallb-suse-edge
```

Comment:
Do we need this? Both the fleet repo and GitRepo's spec point to suse-edge.

Reply:
It is just the gitrepo name. For other SUSE Edge charts, we've already used <product/project>-suse-edge (see https://github.com/suse-edge/misc/blob/main/fleet-examples/gitrepos/kubevirt-suse-edge-gitrepo.yaml#L4 for example)